lib/matplotlib/backends/backend_wx.py

```diff
@@ -1146,6 +1149,31 @@ def __init__(self, canvas):
     zoomStartX = cbook._deprecate_privatize_attribute("3.3")
     zoomStartY = cbook._deprecate_privatize_attribute("3.3")
 
+    @staticmethod
+    def _icon(name):
+        """
+        Construct a `wx.Bitmap` suitable for use as icon from an image file
+        *name*, including the extension and relative to Matplotlib's "images"
+        data directory.
+        """
+        image = np.array(PIL.Image.open(cbook._get_data_path("images", name)))
+        try:
+            dark = wx.SystemSettings.GetAppearance().IsDark()
+        except AttributeError:  # wxpython < 4.1
+            # copied from wx's IsUsingDarkBackground / GetLuminance.
+            bg = wx.SystemSettings.GetColour(wx.SYS_COLOUR_WINDOW)
+            fg = wx.SystemSettings.GetColour(wx.SYS_COLOUR_WINDOWTEXT)
+            # See wx.Colour.GetLuminance.
+            bg_lum = (.299 * bg.red + .587 * bg.green + .114 * bg.blue) / 255
+            fg_lum = (.299 * fg.red + .587 * fg.green + .114 * fg.blue) / 255
+            dark = fg_lum - bg_lum > .2
+        if dark:
+            fg = wx.SystemSettings.GetColour(wx.SYS_COLOUR_WINDOWTEXT)
+            image[(image[..., :3] == 0).all(axis=-1), :3] = (
+                fg.Red(), fg.Green(), fg.Blue())
```

Comment:
```suggestion
            black_mask = (image[..., :3] == 0).all(axis=-1)
            image[black_mask, :3] = fg.Red(), fg.Green(), fg.Blue()
```
for better readability

Reply:
yup